Submodule components/lvgl_esp32_drivers contains modified content
diff --git i/components/lvgl_esp32_drivers/lvgl_helpers.h w/components/lvgl_esp32_drivers/lvgl_helpers.h
index 081a843..728e9b1 100644
--- i/components/lvgl_esp32_drivers/lvgl_helpers.h
+++ w/components/lvgl_esp32_drivers/lvgl_helpers.h
@@ -36,6 +36,15 @@ extern "C" {
 #if defined (CONFIG_CUSTOM_DISPLAY_BUFFER_SIZE)
 #define DISP_BUF_SIZE   CONFIG_CUSTOM_DISPLAY_BUFFER_BYTES
 #else
+
+#ifndef	LV_HOR_RES_MAX
+#define	LV_HOR_RES_MAX	CONFIG_LV_HOR_RES_MAX
+#endif
+
+#ifndef	LV_VER_RES_MAX
+#define	LV_VER_RES_MAX	CONFIG_LV_VER_RES_MAX
+#endif
+
 #if defined (CONFIG_LV_TFT_DISPLAY_CONTROLLER_ST7789)
 #define DISP_BUF_SIZE  (LV_HOR_RES_MAX * 40)
 #elif defined CONFIG_LV_TFT_DISPLAY_CONTROLLER_ST7735S
